fiaas_deploy_daemon/deployer/kubernetes/adapter.py

```diff
@@ -56,12 +59,14 @@ def deploy(self, app_spec: AppSpec):
         self._ingress_deployer.deploy(app_spec, labels)
         self._deployment_deployer.deploy(app_spec, selector, labels, _besteffort_qos_is_required(app_spec))
         self._autoscaler_deployer.deploy(app_spec, labels)
+        self._pod_disruption_budget_deployer.deploy(app_spec, selector, labels)
 
     def delete(self, app_spec: AppSpec):
         self._ingress_deployer.delete(app_spec)
         self._autoscaler_deployer.delete(app_spec)
         self._service_deployer.delete(app_spec)
         self._deployment_deployer.delete(app_spec)
+        self._pod_disruption_budget_deployer.delete(app_spec)
```

Comment:
This is only called on delete watch events (Application resource was deleted). It shouldn't be necessary to add the delete call here since owner references should take care of cleaning up the managed resources. 
(This behavior is left over from before owner references were set on the managed resources, and should probably be cleaned up for other the resources as well, but that's not in scope here )